src/main/java/seedu/address/model/person/LoyaltyPoints.java

```diff
@@ -26,6 +26,26 @@ public LoyaltyPoints(int points) {
         this.value = points;
     }
 
+    /**
+     * Returns true if a given int is a valid loyalty points.
+     */
+    public static boolean isValidLoyaltyPoints(int points) {
+        return 0 <= points; //TODO might want to set upper bound
+    }
+
+    /**
+     * Returns true if a given string is a valid loyalty points.
+     */
+    public static boolean isValidLoyaltyPoints(String points) {
+        int integerPoints = 0;
```

Comment:
why you initialise the value?

Reply:
because java warns me if i dont initialise it and try to use it :(

Reply:
can probably reduce the variable scope, something like
try {
    integerPoints = Integer.parseInt(points);
    return integerPoints >= 0;
} catch (....)

not important though, its just not very good practice to initialize a value to not use it